### DISCUSSION TITLE
Animations - what should be possible already

### DISCUSSION THREAD

**Discussion by TopperDEL** [StereoKit/StereoKit#124] [Q&A]
If I wanted animated objects like a lottie animation or a gltf with an included animation - what would already work or do you know of anything that should/would work? Any Framework you know that I could try?

**Comment by maluoi** [comment]
I see there's a number of Lottie tools for C#, but at a casual glance, I think they might be a little tricky to integrate here. If they have a way to output to a d3d texture, we could possibly get it to work! Since this is 2D stuff, I'm curious how you'd want to primarily use this, would this be for small animations sitting on a UI window or button, would this drive the UI itself, or something else?

I also don't currently support animations in gltf right now either, but that is towards the top of my mind now that the cross-platform graphics system is in-place!

I unfortunately don't have any animation framework recommendations off the top of my head. I think my priority here should be getting gltf animations up and running.

**Reply by TopperDEL** [reply]
I was thinking of extracting the images from at Lotti-converted gif as single frames and place them on a Tex. My goal is to place this as a kind of a billboard or above a button, yes.

I'll try this next.
